app/models/site_config.rb

```diff
@@ -19,7 +19,7 @@ class SiteConfig < RailsSettings::Base
   # staff account
   field :staff_user_id, type: :integer, default: 1
   field :default_site_email, type: :string, default: "yo@dev.to"
-  field :social_networks_handle, type: :string, default: "thepracticaldev"
+  field :social_media_handles, type: :text
```

Comment:
Have you checked [`rails-settings-cached` `type: :hash`](https://github.com/huacnlee/rails-settings-cached#readme) ? It could cover functionality, so that we don't have to reinvent it in our code.

Reply:
I just looked through it now. This is ideal @anna, let me implement and then I'll ping you.

Reply:
Sorry for not mentioning in our previous discussion, it didn't come into my mind that there could be such a feature before reviewing the pr.